OpenChange/iCalEvent+MAPIStore.m

```diff
@@ -457,7 +460,14 @@ - (void) updateFromMAPIProperties: (NSDictionary *) properties
                     {
                       [person setParticipationStatus: newPartStat];
                       [person setRsvp: @"TRUE"];
-                      [person setRole: @"REQ-PARTICIPANT"];
+                      if (i < ccFirstIndex)
```

Comment:
:+1: to @kamenim suggestion. Indeed it is more necessary if there are more changes coming...